### ISSUE TITLE
lux-widget==0.1.6 is broken on JupyterLab

### ISSUE THREAD

**Issue by ChrisCarini** [lux-org/lux-widget#94]
Hello - 

## TL;DR

After a day of troubleshooting, I believe that `lux-widget==0.1.6` as [released on PyPi](https://pypi.org/project/lux-widget/0.1.6/) is broken for JupyterLab in its current state because it was released without this commit -> https://github.com/lux-org/lux-widget/commit/b4a479c015a360b0f52bc754a5dc5841e7195209

## Symptoms

When I click on the `Toggle Pandas/Lux` button, I get either `Error displaying widget: model not found` or `Loading widget...`

## Details

The errors I get in my Chrome console are:

> Uncaught (in promise) Error: Module luxwidget, semver range 0.1.5 is not registered as a widget module

Now, this seemed suspect that it was showing `0.1.5` when I had `0.1.6` installed (either as a dependency pulled in from `lux-api==0.3.1`, or directly specifying `lux-widget==0.1.6` directly.

If I edit both of the below files, change `0.1.5` to `0.1.6`, then run `jupyter lab clean; jupyter lab build --no-minimize` and hard-refresh my browser, the Lux widget loads.
- `/opt/conda/share/jupyter/nbextensions/luxwidget/package.json`
- `/opt/conda/lib/python3.8/site-packages/luxwidget/nbextension/package.json`

It seems that these two files do not have the correct version specified, and this can also be seen in the git history at the specific tagged version for this file:
- https://github.com/lux-org/lux-widget/blob/v0.1.6/luxwidget/nbextension/package.json

@dorisjlee - since you seem to be the main maintainer of this repo, and the one that , would you be able to either re-release version `0.1.6` to PyPi with the above change included, or perhaps version bump and mark `0.1.6` for deprecation? Perhaps there is some other option that I am not familiar with (the implementation and distribution of these extensions is out of my expertise, unfortunately) that you can think of?

**Comment by ChrisCarini** [comment]
I can also confirm that running `pip install git+https://github.com/lux-org/lux-widget.git@b4a479c015a360b0f52bc754a5dc5841e7195209` (that hash is for the commit that changed the version after `0.1.6` was released) works fine on my installation.

**Comment by dorisjlee** [comment]
Hi @ChrisCarini, Thanks for reporting this issue and suggesting this fix! I've bumped up the version to `0.1.7` and ensured that the `package.json` versions are all up to date in the latest release. Let me know if this fixes the issue on your end!

**Comment by ChrisCarini** [comment]
Hi @dorisjlee - absolutely! I can confirm this is now fixed for me! Thank you for the fast turn around!
